test/snapshot-test.js

```diff
@@ -5,7 +5,7 @@
 
 var delayed = require('delayed')
 var common = require('./common')
-var SlowStream = require('slow-stream')
+var trickle = require('trickle')
```

Comment:
@vweevers I take it you ran into some issues with `slow-stream`?

Reply:
TIL about trickle!

Reply:
> I take it you ran into some issues with slow-stream?

`slow-stream` uses `process.hrtime` and `setTimeout`, `trickle` only uses `setInterval`.